monitoring/prometheus/alerts/ceph_default_alerts.yml

```diff
@@ -214,6 +214,17 @@ groups:
             will be full in less than 5 days assuming the average fill-up
             rate of the past 48 hours.
 
+      - alert: MTU Mismatch
+        expr: node_network_mtu_bytes{device!="lo"} != topk(1, node_network_mtu_bytes{device!="lo"}) by (device)
```

Comment:
My bad the `topk()` doesn't do what I thought. So let's go back to the `quantile(0.5,...)`. I tested those in [this page](https://demo.promlens.com/?l=OTuO7josUeU) and it works as expected.

BTW, good idea to exclude `lo` interface. We should test this expression in a real cluster if possible.

Reply:
I tested  `quantile(0.5,..)` function with other values as well(e.g. ceph_mgr_module_status), and most of the times it gave the desired result. IMO as well, quantile is the best approach to deal with this scenario.